radixdlt-core/radixdlt/src/integration/java/com/radixdlt/harness/deterministic/DeterministicActorsTest.java

```diff
@@ -149,19 +149,16 @@ protected void setActorConfigurations(List<ActorConfiguration> actorConfiguratio
   public void setup() {
     this.network =
         new DeterministicNetwork(
-            nodeKeys.stream()
-                .map(k -> BFTNode.create(k.getPublicKey()))
-                .collect(Collectors.toList()),
+            nodeKeys.stream().map(k -> BFTNode.create(k.getPublicKey())).toList(),
             MessageSelector.firstSelector(),
             MessageMutator.nothing());
 
-    List<BFTNode> allNodes =
-        nodeKeys.stream().map(k -> BFTNode.create(k.getPublicKey())).collect(Collectors.toList());
+    List<BFTNode> allNodes = nodeKeys.stream().map(k -> BFTNode.create(k.getPublicKey())).toList();
```

Comment:
is there a repetition between line 152 and 156?

Reply:
Indeed. Good catch.
Fixed.